docs/modules/ROOT/pages/get-started.adoc

```diff
@@ -1,9 +1,246 @@
+== Step 4. Switch Between Clusters
+Having separate local configurations for your development and production clusters allows you to switch between them without leaving the command line. Make sure both clusters are running before you start.
+
+. From the command line, change into `$HOME/.local/share/hz-cli`.
+. Execute the following command to use the default configuration to connect to your development cluster:
++
+```bash
+hzc
+```
++
+As before, the CLC connects to your development cluster.
+. Type `exit` to return to `$HOME/.local/share/hz-cli`.
+. Execute the following command to use the new configuration file for your production cluster.
++
+```bash
+hzc -c <config_prod>.yaml
+```
++
+The CLC connects to your production cluster
+
+[[step-5-write-data]]
+== Step 5. Write Data to a Map
+Now that you've connected to both your clusters, try using the CLC to write data to a map on your development cluster. A quick and easy way to do this is to use a JSON file.
+
+. Copy the following JSON to a file in the directory `$HOME/.local/share/hz-cli`.
++
+.currencycodes.json
+```json
+{
+  "1": {
+        "Code": "CAD",
+        "Currency": "Canadian Dollar"
+      },
+  "2": {
+        "Code": "INR",
+        "Currency": "Indian Rupee"
+      },
+  "3": {
+        "Code": "MXN",
+        "Currency": "Mexican Peso"
+      },
+  "4": {
+        "Code": "GBP",
+        "Currency": "Pounds Sterling"
+      },
+  "5": {
+        "Code": "TRY",
+        "Currency": "Turkish Lira"
+      },
+  "6": {
+        "Code": "USD",
+        "Currency": "United States Dollar"
+      },
+}
+
+```
+. As before, from the command line change into `$HOME/.local/share/hz-cli`. This time execute a 'put-all' command to write the data from the JSON file to a new map called `currency`.
++
+```bash
+hzc map put-all -n currency --json-entry currencycodes.json
+```
++
+The `-n` flag passes the map name to your cluster.
+
+Do a quick check on your cluster to make sure that your data has written successfully.
+.	Open the dashboard of the development cluster and select *Manage Cluster* > *Management Center*.
+.	Go to *Storage* > *Maps*. You'll see that your cluster has a map called `currency` with six entries. 
+
+
+[[step-6-query-map]]
+== Step 6. Query Map Data
+You can use SQL to query the data in your `currency` map.
+
+.	Start by creating a mapping to the `currency` map.
+
++
+```bash
+hzc sql "CREATE MAPPING currency (__key INT, code VARCHAR, currency VARCHAR) TYPE IMap OPTIONS('keyFormat'='int', 'valueFormat'='json-flat')"
+```
+The SQL mapping statement does a number of things:
+
+** Adds column headings for currencies and codes
+** Creates a SQL connection to the map
+** Tells Hazelcast how to serialize and deserialize the keys and values.
+
+. Try running some simple queries against the `currency` map. For example, this query returns all data in the map and orders it by the currency code.  
++
+```bash
+hzc sql "select * from currency order by code"
```

Comment:
```suggestion
hzc sql "SELECT * FROM currency ORDER BY code"
```

Reply:
will need to check the SQL queries for correctness.

Reply:
Updated as requested.